vulture/core.py

```diff
@@ -116,6 +116,17 @@ def size(self):
         assert self.last_lineno >= self.first_lineno
         return self.last_lineno - self.first_lineno + 1
 
+    def get_report(self, sort_by_size=False):
+        if sort_by_size:
+            line_format = 'line' if self.size == 1 else 'lines'
+            size_report = ', {0:d} {1}'.format(self.size, line_format)
+        else:
+            size_report = ''
+        message = "{0}:{1:d}: {2} ({3}% confidence{4})".format(
```

Comment:
no need for `message` variable, just return it.

Reply:
Done!